README.md

```diff
@@ -98,6 +101,63 @@ yum-config-manager \
     https://download.docker.com/linux/centos/docker-ce.repo
 yum update && yum install docker-ce-18.06.1.ce
 ```
+
+KubeEdge's Cloud(edgecontroller) connects to Kubernetes master to sync updates of node/pod status. If you don't have Kubernetes setup, please follow these steps to install Kubernetes using kubeadm
+
+#### Install kubeadm/kubectl 
+
+For Ubuntu:
```

Comment:
kubeadm/kubectl install instructions can be redirected to https://kubernetes.io/docs/setup/independent/install-kubeadm/.
we don't need to repeat the instructions here again.

Reply:
We have tried to keep a constant look in the as we give installation steps for docker and mosquitto as well. We will fix these in future PR's